tests/utils/test_storage_utils.py

```diff
@@ -77,13 +77,13 @@ def test_minio_connectivity(minio: Minio):
     assert not minio_object_exists(minio, bucket, 'another/file.txt')
 
 
-async def test_minio_push_empty(storage: MinioStorage, bucket, job_dir, caplog):
+async def test_minio_push_empty(storage: MinioStorage, bucket, job_dir, caplog, minio):
```

Comment:
why it is needed to use the `minio` fixture explicitly?

Reply:
possibly a result of some experimentation when CI was failing... will investigate.